### ISSUE TITLE
Configuration UI for settings import and export

### ISSUE THREAD

**Issue by garethbowen** [medic/medic-webapp#3868]
Provide a UI for exporting and importing settings so we can stop using the Dashboard app settings screen. Also publish an API so import/export is easily scriptable. Consider moving the update_settings implementation in the database into the API layer.

**Comment by amandacilek** [comment]
Summary of proposal which I think we all agree on:
- Add a third tab to the Settings page called “Backup/Restore”
- This page should include a sentence or two of text instructions with two buttons - one for Download and one for Upload

![settings_importexport](https://user-images.githubusercontent.com/26235957/32284356-3d33f79c-bee4-11e7-914c-c7ee1a84080f.png)

Note from Marc: "Our new config for which we use medic-config is a whole directory of things: settings, forms, icons, translations, etc. We may support uploading that all together at some point, perhaps as a zip file. That could go in elsewhere, perhaps import/export, since it covers several config tabs worth of data. I guess it will be a separate ticket for download/upload of the entire config."

**Comment by alxndrsn** [comment]
@garethbowen would this be a good place to start on moving the admin interface into a separate app?

**Comment by garethbowen** [comment]
> would this be a good place to start on moving the admin interface into a separate app?

To keep it as small as possible I want to just to a 1-to-1 migration from the admin tab to the admin app. I guess this is blocked on that refactor happening.

> Is this already available at /api/v1/settings?

Absolutely. Good spotting!

**Comment by garethbowen** [comment]
No longer blocked...

**Comment by garethbowen** [comment]
Code review please @dianabarsan

**Comment by ngaruko** [comment]
Getting errors on [alpha-old](https://alpha-old.dev.medicmobile.org/medic/_design/medic-admin/_rewrite/#/settings/backup) (`alpha.dev` has issues upgrading) when uploading new settings:
![image](https://user-images.githubusercontent.com/6979995/39972389-79109628-5762-11e8-87be-dadbdf652879.png)

**Comment by garethbowen** [comment]
@ngaruko I think this is the old problem with api and sentinel not being upgraded. Please delete these directories and try upgrading again.

**Comment by ngaruko** [comment]
LGTM
